tap_square/client.py

```diff
@@ -116,7 +115,33 @@ def get_locations(self):
 
             yield (result.body.get('locations', []), result.body.get('cursor'))
 
-    def get_refunds(self, object_type, start_time, bookmarked_cursor):  # TODO:check sort_order input
+    def get_inventories(self, variation_ids, start_time):
+        with singer.http_request_timer('GET inventories'):
+            result = self._client.inventory.batch_retrieve_inventory_counts(body={
+                "catalog_object_ids": variation_ids,
+                "updated_after": start_time,
+            })
+
+        if result.is_error():
+            raise Exception(result.errors)
+
+        yield (result.body.get('counts', []), result.body.get('cursor'))
+
+        # Not sure if the pagination works like this. I expect this to fail like payments and refunds did
```

Comment:
@kspeer825 as I said in slack, I have a feeling the pagination will also not work for this stream... there's probably a general pattern we have to fix for all of them.